SingularityBase/src/main/java/com/hubspot/singularity/SingularityLimits.java

```diff
@@ -0,0 +1,13 @@
+package com.hubspot.singularity;
+
+public class SingularityLimits {
+  private final int maxDecommissioningAgents;
+
+  public SingularityLimits(int maxDecommissioningAgents) {
+    this.maxDecommissioningAgents = maxDecommissioningAgents;
+  }
```

Comment:
Since we don't have immutables set up in Singularity, this needs a `@JsonCreator` annotation on the constructor and @JsonProperty("maxDecommissioningAgents")` on the field. See other things in the Base module for examples